src/RegistrationController.js

```diff
@@ -99,20 +127,38 @@ function (Okta, BaseLoginController, FormType, Enums, BaseSchema, SchemaFormFact
         }
       });
       this.model = new Model();
-      
+
+      var checkPasswordMeetComplexities = function(model) {
+        var password = model.get('password') || '';
+        _.each(schema.passwordComplexity.enabledComplexities, function(complexityName) {
+          var ele = Okta.$('.password-complexity-' + complexityName);
+          var complexityValue = this.get(complexityName);
+          if (PasswordComplexityUtil.complexities[complexityName].doesComplexityMeet(complexityValue, password, model)){
+            ele.addClass('password-complexity-meet');
+          } else {
+            ele.removeClass('password-complexity-meet');
+          }
+        }, schema.passwordComplexity);
+      };
+
       var form = new Form(this.toJSON());
       properties.each(function(schemaProperty) {
-        form.addInput(_.extend(SchemaFormFactory.createInputOptions(schemaProperty), {
-          label: false,
-          'label-top': true,
-          placeholder: schemaProperty.get('description')
-        }));
+        var inputOptions = RegistrationFormFactory.createInputOptions(schemaProperty);
+        var name = schemaProperty.get('name');
+        if (name === 'password' || name === 'login') {
```

Comment:
curiosity: login is the email input correct? so, why we need to check password meet complexity also when the user input in email field?